copter/source/docs/additional-information.rst

```diff
@@ -0,0 +1,17 @@
+.. _additional-information:
+
+=============================================
+Additional Information and Topics of Interest
+=============================================
+
+
+.. toctree::
+    :maxdepth: 1
+
+    Upcoming Features <common-master-features>
+    Antenna Tracking <common-antenna-tracking>
```

Comment:
I think we can lose the Antenna Tracking pages (eventually) if we move this high level info to the Antenna tracker page.

Reply:
will put on the list